features/restaurant_edit_details.feature

```diff
@@ -0,0 +1,21 @@
+Feature: As a system user
+  In order to keep information about my restaurant up to date
+  I would like to be able to edit my restaurant's details
+
+  Background:
+    Given I am logged in as a user of the system
+    And the following restaurant exists
+    | id | name    | address     | zip_code |
+    | 1  | Mikey's | Main Street | 12345    |
```

Comment:
There is no need to set the `id` in this table. You dont use it for anything. Less is more ;-)